test/powershell/Modules/Microsoft.PowerShell.Core/Out-Host.Tests.ps1

```diff
@@ -27,8 +21,9 @@ Describe "Out-Host Tests" -tag CI {
     }
     It "Out-Host writes to host output" {
         $stringToWrite = "thing to write"
+        $stringExpected = "::$($stringToWrite):NewLine"
```

Comment:
No, it's the colon at `:NewLine`. Maybe use another special character as the delimiter?